pkg/controller/master/upgrade/upgrade_controller.go

```diff
@@ -527,6 +531,33 @@ func (h *upgradeHandler) cleanup(upgrade *harvesterv1.Upgrade, cleanJobs bool) e
 		}
 	}
 
+	return h.resumeManagedCharts()
+}
+
+func (h *upgradeHandler) resumeManagedCharts() error {
+	managedCharts, err := h.managedChartCache.List(util.FleetLocalNamespaceName, labels.Everything())
+	if err != nil {
+		return nil
+	}
+
+	// those managedcharts might be paused by the upgrade script, resume them if they are not un-paused
+	targetManagedcharts := map[string]struct{}{util.HarvesterCRDManagedChart: {}, util.HarvesterManagedChart: {}, util.RancherLoggingCRDManagedChart: {}, util.RancherMonitoringCRDManagedChart: {}}
+
+	for _, managedChart := range managedCharts {
+		if !managedChart.Spec.Paused {
+			continue
+		}
+		if _, ok := targetManagedcharts[managedChart.Name]; !ok {
+			continue
+		}
+		mc := managedChart.DeepCopy()
+		mc.Spec.Paused = true
```

Comment:
I think this is `false`? So we can resume managed charts?

Reply:
super, it definitely should be `false` here